src/components/LessonList/components/PaginatedLessonList/components/LessonTeaser/index.js

```diff
@@ -35,14 +34,14 @@ export default ({instructor, lesson}) => {
         }}>
           {lesson.summary
             ? <Markdown>
-                {truncate(lesson.summary, {length: 144})}
+                {lesson.summary}
               </Markdown>
-            : '...'
+            : null
           }
         </div>
         <div className='mt2'>
           <Link to={`/lessons/${lesson.slug}`}>
-            <Button size='small'>
+            <Button size='extra-small'>
```

Comment:
Sounds good. Would like design to review as you said, because if we are usually using small buttons the default designs and egghead-ui component should reflect that so default becomes smaller or something.